python/python/tests/test_fragment.py

```diff
@@ -23,6 +24,15 @@
 from lance.fragment import write_fragments
 from lance.progress import FileSystemFragmentWriteProgress
 
+CONFIG = {
+    "allow_http": "true",
+    "aws_access_key_id": "ACCESSKEY",
+    "aws_secret_access_key": "SECRETKEY",
+    "aws_endpoint": "http://localhost:9000",
+    "dynamodb_endpoint": "http://localhost:8000",
+    "aws_region": "us-west-2",
+}
```

Comment:
Please don't put this test in `test_fragment.py`. I'd like to keep all the integration tests in [lance/python/python/tests/test_s3_ddb.py](https://github.com/lancedb/lance/blob/d79e870628b8d5f78cf018e6fd184c503c364b1e/python/python/tests/test_s3_ddb.py#L230-L243)

Reply:
moved.